pkg/flare/archive.go

```diff
@@ -173,6 +173,12 @@ func createArchive(confSearchPaths SearchPaths, local bool, zipFilePath string,
 		hostname = "unknown"
 	}
 
+	logLevel := config.Datadog.GetString("log_level")
+	if len(logLevel) != 0 {
+		delim := "-"
+		hostname = strings.ToUpper(logLevel) + delim + hostname
```

Comment:
Hmm - why do we want the hostname to be prefixed with the log level? Wouldn't the archive name already contain this? I'm also not fully sure but I think this would make debugging harder as the hostname would not be the actual hostname of the machine being looked at, causing confusion.

Reply:
The prefix at this location would be marginally useful once the zip had been opened, however, agreed, it isn't worth the potential issues or confusion.